packages/application/src/shell.ts

```diff
@@ -412,6 +412,13 @@ export class LabShell extends Widget implements JupyterFrontEnd.IShell {
     hboxPanel.addWidget(vsplitPanel);
     hboxPanel.addWidget(rightHandler.sideBar);
 
+    const scanModeOffRegion = document.createElement('div');
+    scanModeOffRegion.setAttribute('aria-live', 'assertive');
+    scanModeOffRegion.setAttribute('role', 'region');
+    scanModeOffRegion.setAttribute('id', 'screen-reader-aria-live');
+    scanModeOffRegion.className = 'jp-ScanModepanel-ariaLive';
+    hboxPanel.node.appendChild(scanModeOffRegion);
```

Comment:
This is appending a child to a managed layout outside of the supported `addWidget` method; it might have side-effects, and the node might be lost on refresh. Does it need to be added to hboxPanel? Can it be added directly to body?

Reply:
We have followed how skip links is rendered (as a widget) and implemented scan mode off that same way.